docs/guide/authjs/quick-start.md

```diff
@@ -0,0 +1,152 @@
+
+```ts
+import { NuxtAuthHandler } from '#auth'
+
+export default NuxtAuthHandler({
+  // your authentication configuration here!
+})
+```
+
+### Adding a provider
+
+After creating your NuxtAuthHandler, you can begin by adding a provider. You can find an overview of all the avalible providers [here](https://next-auth.js.org/providers/). For this example we will add the GitHub provider.
+
+```ts
+import GithubProvider from 'next-auth/providers/github'
+import { NuxtAuthHandler } from '#auth'
+
+export default NuxtAuthHandler({
+  // A secret string you define, to ensure correct encryption
+  secret: 'your-secret-here',
+  providers: [
+    // @ts-expect-error Use .default here for it to work during SSR.
+    GithubProvider.default({
+      clientId: 'your-client-id',
+      clientSecret: 'your-client-secret'
+    })
+  ]
+})
+```
+
+:::warning
+After importing your provider from `next-auth/providers/[PROVIDER]`, call it using `.default()` inside of the providers array configuration. This is required for SSR to properly function.
+:::
+
+The NuxtAuthHandler accepts [all options that NextAuth.js](https://next-auth.js.org/configuration/options#options) accepts for its API initialization. Use this place to configure authentication providers (oauth-Google, credential flow, ...), your secret, add callbacks for authentication events, configure a custom logger and more. Read the [NextAuth.js docs](https://next-auth.js.org/configuration/options#options) to see all possible options.
+
+### Setting a secret
+
+In addition to setting a provider, you also need to set a `secret` which is used to encrypt the JWT Tokens. To avoid hard-coding of the secret you can make it configurable at runtime by using an environment variable and exporting it at runtime or by using the [Nuxt `useRuntimeConfig`](https://nuxt.com/docs/api/composables/use-runtime-config) (and then also setting the correct environment at runtime):
+
+```ts
+import { NuxtAuthHandler } from '#auth'
+
+export default NuxtAuthHandler({
+  // A secret string you define, to ensure correct encryption - NUXT_AUTH_SECRET required in production
+  secret: useRuntimeConfig().authSecret
+
+  // rest of your authentication configuration here!
+})
+```
+
+::: details Full code
+```ts
+// file: ~/server/api/auth/[...].ts
+import GithubProvider from 'next-auth/providers/github'
+import { NuxtAuthHandler } from '#auth'
+
+export default NuxtAuthHandler({
+  secret: useRuntimeConfig().authSecret,
+  providers: [
+    // @ts-expect-error Use .default here for it to work during SSR.
+    GithubProvider.default({
+      clientId: 'your-client-id',
+      clientSecret: 'your-client-secret'
+    })
+  ]
+})
+```
+:::
+
+## Next Steps
+
+Congrats! You have now configured your first authention provider and can login to the application! We recommend the following next steps to continue configuring your application:
+
+TODO: Add next steps here!
```

Comment:
Needs addressing or removing :slightly_smiling_face:

Reply:
Done, added the next steps! I had written this guide before anything else, which is why I still had a placeholder 😊